draft-irtf-cfrg-opaque.md

```diff
@@ -749,17 +772,20 @@ The registration protocol then runs as shown below:
                         response
               <-------------------------
 
- (record, export_key) = FinalizeRequest(client_private_key,
-                                        password,
-                                        blind,
-                                        response,
+ (record, export_key) = FinalizeRequest(response,
                                         server_identity,
                                         client_identity)
 
                         record
               ------------------------->
 ~~~
 
+The client may utilize a state structure `RegistrationState` with the following
```

Comment:
Can we revert this state change? It's separate from the AKE abstraction changes and (I think) sort of complicates presentation now.

Reply:
Do you prefer it in another PR or totally absent?

Reply:
Right now, I lean towards it being totally absent, but we can always re-assess in a followup PR.

Reply:
Alright. I need to adapt the API to ask for the arguments that would otherwise be in the state

Reply:
Reverted 👍